packages/next-templates/src/app/styling/css/productpagina-klachtenformulier.css

```diff
@@ -0,0 +1,8 @@
+/* BREADCRUMNAV */
+.breadcrumb-nav {
+  padding: var(--voorbeeld-space-block-rabbit);
+}
+
+.icon-chevron-right {
+  color: var(--voorbeeld-color-violet-700);
```

Comment:
Gebruik graag een common token voor de iconen-kleur (decoration color ofzo) zodat het icoon kan meeveranderen in de theme builder (https://frameless.studio/)

Reply:
Als dit bedoeld is om de breadcrumb chevron een kleur te geven, zou dat niet gewoon kunnen door de breadcrumb separator color token in te stellen?

Reply:
Dit moet eerst ingesteld worden voor GV. 

Voor nu sluit ik deze convo en maak een issue aan dat al dit soort hard coded styling aangepast moet worden.